apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -289,7 +273,9 @@ NetSimLocalClientNode.prototype.disconnectRemote = function (onComplete) {
 
     self.myWire = null;
     // Trigger an immediate router update so its connection count is correct.
-    self.myRouter.update(onComplete);
+    self.myRouter.update(function (err, result) {
+      onComplete(result);
+    });
```

Comment:
I figured I'd get the rest of these one-off cases as I went in future updates; I just wanted to change the core stuff now, to establish a baseline.  I can finish changing the remaining callbacks now, if you think it's worth doing.